apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -58,6 +59,12 @@ public sealed class PooledSession : IDisposable
         /// </summary>
         internal ModeOneofCase TransactionMode { get; }
 
+        /// <summary>
+        /// The read timestamp of the transaction. (Always <c>null</c> if and
```

Comment:
Is this really and `if and only if`? I think it may also be null if ReturnReadTimestamp was set to false.

Reply:
You're totally right. I updated the text.